ext/cl_khr_external_memory.asciidoc

```diff
@@ -181,10 +182,15 @@ Following new enums are added to the list of supported _param_names_ by {clGetDe
 | {CL_DEVICE_EXTERNAL_MEMORY_IMPORT_HANDLE_TYPES_KHR}
   | {cl_external_memory_handle_type_khr_TYPE}[]
       | Returns the list of importable external memory handle types supported by _device_.
+| {CL_DEVICE_EXTERNAL_MEMORY_IMPORT_ASSUME_LINEAR_HANDLE_TYPES_KHR}
+  | {cl_external_memory_handle_type_khr_TYPE}[]
+      | Returns the list of importable external memory handle types supported by _device_, that are assumed to have linear layout when no other layout information is provided.
 |====
 
 {clGetDeviceInfo} when called with param_name {CL_DEVICE_EXTERNAL_MEMORY_IMPORT_HANDLE_TYPES_KHR} must return a non-empty list of external memory handle types for at least one of the devices in the platform.
 
+{clGetDeviceInfo} when called with param_name {CL_DEVICE_EXTERNAL_MEMORY_IMPORT_ASSUME_LINEAR_HANDLE_TYPES_KHR} returns a list of external memory handle types that are assumed to have a linear memory layout.  This list contains a subset of {CL_DEVICE_EXTERNAL_MEMORY_IMPORT_HANDLE_TYPES_KHR}.  The returned list may be empty.  External memory handle types not in {CL_DEVICE_EXTERNAL_MEMORY_IMPORT_ASSUME_LINEAR_HANDLE_TYPES_KHR} may have any memory layout.
+
```

Comment:
I suggest we move all of this to the table and clearly state that the layout assumed or inferred by the implementation when it cannot assume linear is implementation-defined.

We might also want to clearly state that the only scenarios that are 100% guaranteed to work are those using linearly-tiled images.